### PR TITLE
Bump to latest Checkra1n version and update to 1.1.7

### PR DESCRIPTION
**Pull request by TeGaX**

### CODE REVIEW

**Comment by Mehrshadfi** [comment]
Hi
When will the next update come?

**Comment by TeGaX** [comment]
> Hi
> When will the next update come?

Idk really
Waiting for the dev merge my commit and build it

**Comment by TeGaX** [comment]
> Hi
> When will the next update come?

Or you can check my Checkn1x repo https://github.com/TeGaX/checkn1x_reborn

It updated to latest Odysseyn1x and Checkra1n

**Comment by noelff** [comment]
image

**Comment by realhot** [comment]
image

**Comment by faheem123456** [comment]
iOS 14.5 checkr1n iPhone 8 global not supported plz add iPhone 8 global

**Comment by faheem123456** [comment]
> No description provided.

iPhone 8 global iOS 14.5 not supported

File: README.md [hunk 1]
```diff
@@ -1,23 +1,26 @@
 ![](https://github.com/asineth0/checkn1x/blob/master/icon_dark.png?raw=true)
 
-# checkn1x
+# checkn1x Reborn - bring back Checkn1x with latest Checkra1n version (0.12.3)
```
Comment (Favna) [inline README.md:3]: You should probably remove this from the PR. It’s fine to have it on your own fork, but then create the PR from another branch at least.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,26 @@
...
 
 Linux-based distribution for jailbreaking iOS devices w/ checkra1n.
+Full latest changelogs at [here](https://checkra.in/releases/0.12.3-beta)
 
 ## Downloads
 
-Downloads are available under [releases](https://github.com/asineth0/checkn1x/releases).
+Downloads are available under [releases](https://github.com/TeGaX/checkn1x/releases).
```
Comment (Favna) [inline README.md:10]: Revert this url

File: README.md [hunk 3]
```diff
@@ -1,23 +1,26 @@
...
 
 ## Usage
 
 **Use whatever tool you want, but I'm only officially supporting Etcher.**
 
-1. Download [Etcher](https://etcher.io) and the ISO from releases.
+1. Download [Etcher](https://etcher.io), [Rufus](https://rufus.ie) and the ISO from releases.
```
Comment (Favna) [inline README.md:16]: Etcher works fine, Rufus is not needed. Also Rufus is windows only whereas etcher is cross platform.

File: README.md [hunk 4]
```diff
@@ -29,6 +32,8 @@ apt install curl ca-certificates tar gzip grub2-common grub-pc-bin grub-efi-amd6
 
 # archlinux
 pacman -S --needed curl tar gzip grub mtools xorriso cpio xz
-
 sudo ./build.sh
 ```
+
+## Credits
```
Comment (Favna) [inline README.md:38]: Remove this

File: scripts/checkn1x_cliprompt [hunk 5]
```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 echo 'Press enter to run checkra1n in CLI mode.'
 read -s
-exec /usr/local/bin/checkra1n -cv
+exec /usr/local/bin/checkra1n --cli
```
Comment (Favna) [inline scripts/checkn1x_cliprompt:4]: Correct me if I’m wrong but I’m pretty sure -c is an alias for —cli and -v for —verbose. If that’s correct (validate with —help) then revert this as the previous flags were fine.

**Review by Favna — CHANGES_REQUESTED** [review]

**Pull request closed without merging**